change/@internal-react-components-5c4e27c8-a1aa-4f57-a5fa-3176b703b9ad.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Remove this file. We are not changing anything in the react-components package anymore

Reply:
Done